### PR TITLE
Add details about sysbench and the SQL Plan Cache

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/11011
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/benchmark-tidb-using-sysbench.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ server_configs:
     log.level: "error"
 ```
 
+同时，推荐启用 [`tidb_enable_prepared_plan_cache`](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，并保证 `--db-ps-mode` 没有设置为 `disabled`，这样 sysbench 就可以使用预处理语句。关于 SQL 执行计划缓存的功能及监控，请参考[执行计划缓存](/sql-plan-cache.md)。
```
Comment (shichun-0415) [inline benchmark/benchmark-tidb-using-sysbench.md:22]:
```suggestion
同时，推荐启用 [`tidb_enable_prepared_plan_cache`](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，并保证 `--db-ps-mode` 没有设置为 `disabled`，这样 sysbench 就可以使用预处理语句。关于 SQL 执行计划缓存的功能及监控，请参考[执行计划缓存](/sql-prepared-plan-cache.md)。
```

File: sql-prepared-plan-cache.md [hunk 2]
```diff
@@ -272,3 +272,9 @@ mysql> select @@last_plan_from_cache;       -- 因为开关打开，第二次依
 +------------------------+
 1 row in set (0.00 sec)
 ```
+
+### 监控
+
+在 [Grafana 面板](/grafana-tidb-dashboard.md)的 TiDB 页面，**Executor** 部分包含 “Queries Using Plan Cache OPS" 和 “Plan Cache Miss OPS” 两个图表，用以检查 TiDB 和应用是否正确配置，以便 SQL Plan Cache 能正常工作。TiDB 页面的 **Server** 部分还提供了 “Prepared Statement Count” 图表，如果应用使用了预处理语句，这个图表会显示非零值，通过数值变化，可以判断 SQL Plan Cache 是否正常工作。
```
Comment (Oreoxmt) [inline sql-prepared-plan-cache.md:278]:
```suggestion
在 [Grafana 面板](/grafana-tidb-dashboard.md)的 TiDB 页面，**Executor** 部分包含“Queries Using Plan Cache OPS”和“Plan Cache Miss OPS”两个图表，用以检查 TiDB 和应用是否正确配置，以便 SQL 执行计划缓存能正常工作。TiDB 页面的 **Server** 部分还提供了“Prepared Statement Count”图表，如果应用使用了预处理语句，这个图表会显示非零值。通过数值变化，可以判断 SQL 执行计划缓存是否正常工作。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**